File: exercicios em C/q15.cpp
#include <stdio.h>

int main(void) {
	int media, soma, n, pontos, i;
	
	printf ("\n\nEste programa determima a media bimestral de uma sala.");
	
	printf ("\n\nDigite a quantidade de alunos: ");
	scanf ("%d", &n);
	
	for (i = 1; i <= n; i++) {
		printf ("\n\nDigite a pontucao do %d aluno: ", i);
		scanf("%d", &pontos);
		soma = pontos + soma;
	}	
	media = soma / n;
	printf("\n\nA media a media bimestral da turma e: %d.", media);
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}

File: exercicios em Python/q15.py
import os
print("\nEste programa determina a media bimestral de uma turma!")
n = int(input("\nDigite a quantidade de alunos pertencentes a turma: "))
i = 1
soma = 0
while i <= n:
    nota = int(input("\nDigite a nota do aluno: "))
    soma = soma + nota
    i = i + 1
    
media = int (soma / n)
print("\n\nA media da turma e de ",media,".")
print("\nObrigado por utilizar nosso sistema!")
os.system ("pause")
        
